templates/discourse/files/config/discourse.platform.sh.conf

```diff
@@ -0,0 +1,26 @@
+env RAILS_ENV=production
+redis_host  = cache.internal
+redis_port  = 6379
+db_host     = database.internal
+db_port     = 5432
+db_name     = main
+db_username = main
+db_password = main
+smtp_address = <%= ENV['PLATFORM_SMTP_HOST']%>
+developer_emails= <%= ENV['DISCOURSE_DEVELOPER_EMAILS']%>
+# smtp authentication mechanism
+smtp_authentication = none
+# enable TLS encryption for smtp connections
+smtp_enable_start_tls = false
+serve_static_assets = true
+config.assets.compile = true
+config.assets.digest = false
+
+# you should export in the environment using the CLI or set here the specifics
+# platform vset -emaster env:DISCOURSE_DEVELOPER_EMAILS myemail@example.com
+# platform vset env:DISCOURSE_MAXMIND_LICENSE_KEY get_a_key_from_maxmind
```

Comment:
These don't seem to do anything...?  The site installs without them.

Reply:
You will very probably have errors if you don't. Possibly they corrected the install process around those two .. but I don't believe so. At any rate it is good to have the "developer email" set to the project admin.